build.cake

```diff
@@ -1,3 +1,9 @@
+#addin "nuget:?package=Cake.Xamarin&version=3.1.0"
+
+Action<string> write = (msg) =>
```

Comment:
Nit-pick: rename write to Write for consistency.

Reply:
Not sure I agree that capitalizing will add to consistency.  `write` is a local variable in this context, similarly named to the other functions below in the same file, for example: https://github.com/okta/okta-oidc-xamarin/pull/16/files/3a43502b3a1f2f66fb55f1ff7c4a8bda21fd4576#diff-4d3634747586b964df9f7712bc3ec669R96